docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
             - SHOW_PROGRESS=False
         volumes:
             - ./data:/data
-        command: bash -c 'echo "https://github.com/pirate/ArchiveBox" | /bin/archive; tail -f /dev/null'
+        command: bash -c 'echo "https://github.com/pirate/ArchiveBox" | /bin/archivebox; tail -f /dev/null'
```

Comment:
I'd rather use `/bin/archive` to stay consistent with the rest of the docs for ArchiveBox usage with the `./archive` shortcut in the project root.